lib/constable_web/controllers/announcement_controller.ex

```diff
@@ -109,6 +109,22 @@ defmodule ConstableWeb.AnnouncementController do
     end
   end
 
+  def delete(conn, %{"id" => id}) do
```

Comment:
What do you think about adding controller tests for this action? Not sure if it's needed or not, but it might be good to have something in lieu of not having feature tests that test this path.